api/cluster.go

```diff
@@ -322,7 +322,9 @@ func policyStatements(workerRole string, identityProvider string, subjectKey str
 func (cluster Cluster) IsOldestReadyCluster() bool {
```

Comment:
_maybe_ we need to change the name as well? :shrug: 

(or roll out a version with an additional method and then remove this one after migrating)

Reply:
Done. Don't want to over-complicate it but need to keep the old name for migration for now.